pkg/util/util.go

```diff
@@ -25,3 +25,13 @@ func StringListEqual(list1, list2 []string) bool {
 
 	return s1.Equal(s2)
 }
+
+// Contains searches if a string list contains the given string or not.
+func Contains(list []string, strToSearch string) bool {
```

Comment:
there is already a defined func with the same functionality:
https://github.com/kubernetes/cloud-provider-openstack/blob/d38032ff1a7da28a0648d8f6e3ac5d3750f1a53f/pkg/autohealing/utils/utils.go#L20

Reply:
Thanks @kayrus, the plan is to move the common utility functions to a single place gradually. That said, we should remove `Contains` function for autohealing and use this one instead, but that's another PR.